manifests/r/reMarkable/reMarkableCompanionApp/3.1.1.722/reMarkable.reMarkableCompanionApp.installer.yaml

```diff
@@ -0,0 +1,14 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: reMarkable.reMarkableCompanionApp
+PackageVersion: 3.1.1.722
+Installers:
+- Architecture: x64
+  InstallerType: exe
+  InstallerUrl: https://downloads.remarkable.com/latest/windows

```

Comment:
I purposefully used the /latest/windows url instead of the version specific url, so that winget can keep this package automatically up to date when /latest/windows returns a new installer hash in the future.